util/paging.go

```diff
@@ -0,0 +1,27 @@
+package util
+
+import (
+	"net/url"
+	"strconv"
+)
+
+// Paging - pagination
+func Paging(query url.Values) (int, int) {
+
+	page, _ := strconv.Atoi(query.Get("page"))     // page number
```

Comment:
what is page parameters is not there

Reply:
If page or limit is not there then default values are taken for limit & offset ( 5, 0)